sonic-chassisd/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
I believe you can remove this warning filter, and instead, enable pytest coverage reporting. You can use this PR as an example: https://github.com/Azure/sonic-platform-common/pull/139/files. Make sure to add `'pytest-cov'` to `tests_require` and also update the .gitignore file in this repo to ignore the generated coverage files.

Reply:
Done. Please review. Attached report below.